grails-app/views/order/show.gsp

```diff
@@ -306,6 +326,21 @@
                 $(currentRow).hide();
               });
             }
+
+            function fetchOrderItemsDerivedStatus() {
```

Comment:
@jmiranda Moving this to async improved the loading/rendering times by another ~2s

Reply:
:+1: